code/modules/plumbing/plumbers/synthesizer.dm

```diff
@@ -123,6 +95,8 @@
 	data["current_reagent"] = ckey(initial(reagent_id.name))
 	return data
 
+//NSV13 Removed need for matter cartridges, need speedy heals pls
+
```

Comment:
Can you revert the changes to this file? I think the solution presented in Karmic's PRs regarding this is better and don't want them to conflict during testmerging.